src/components/LoginDialogueBox.tsx

```diff
@@ -71,6 +71,7 @@ const LoginDialogueBox = (props: DialogueProps) => {
           'volunteerId',
           volunteerData.data.loginVolunteer.volunteer.id,
         );
+        localStorage.setItem('type', 'volunteer');
```

Comment:
instead of the magic string, you might want to use a constant so you don't accidentally make a typo somewhere